model/version.go

```diff
@@ -106,17 +107,29 @@ func (v *Version) LastSuccessful() (*Version, error) {
 	return lastGreen, nil
 }
 
-func (self *Version) UpdateBuildVariants() error {
+func (self *Version) UpdateBuildVariantsAndActivation() error {
 	return VersionUpdateOne(
 		bson.M{VersionIdKey: self.Id},
 		bson.M{
 			"$set": bson.M{
+				VersionIsActiveKey:      true,
 				VersionBuildVariantsKey: self.BuildVariants,
 			},
 		},
 	)
 }
 
+func (self *Version) UpdateActivation() error {
```

Comment:
This should check if version already is set active, otherwise we don't need to set it again (so it should also set the local copy in case we're ever using the same version object many times)

Reply:
sorry, small change I should've thought of already: Can we call this "SetActivated"? otherwise it implies we could be setting it true or false based on what's in the local copy